### PR TITLE
Messletters2

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#101 
#### What does this implement/fix? Explain your changes.
- 20 new fonts added
	1. scammer
	2. strikethrough
	3. drako
	4. celtic
	5. fasion
	6. curly
	7. russian
	8. messletters
	9. swirly
	10. dotted
	11. parenthesized
	12. strange
	13. wiggly
	14. hazy
	15. hyves
	16. awcute
	17. slammer
	18. coptic1
	19. coptic2
	20. fancy108
- `setup.py` modified
- `coverage` dependency moved to `extras_require`
- Test system modified
- `requirements.txt` removed

### CODE REVIEW

File: .travis/install.sh [hunk 1]
```diff
@@ -11,11 +11,11 @@ then
 	 PYTHON_COMMAND=python3
 	 PIP_COMMAND=pip3
 fi
- 
-$PIP_COMMAND install -r requirements.txt
+
 $PYTHON_COMMAND setup.py install
 $PYTHON_COMMAND -m art test
 $PYTHON_COMMAND -m art test2
+$PIP_COMMAND install -e .[Coverage]
```
Comment (sadrasabouri) [inline .travis/install.sh]: What is this whole commit for?
Can you please explain the reason?

File: .travis/test.sh [hunk 2]
```diff
@@ -29,4 +31,6 @@ then
 fi
 
 
-$PYTHON_COMMAND -m cProfile -s cumtime art_profile.py
\ No newline at end of file
+$PYTHON_COMMAND -m cProfile -s cumtime art_profile.py
+$PIP_COMMAND uninstall art
+$PIP_COMMAND install art --upgrade
```
Comment (sadrasabouri) [inline .travis/test.sh]: new line missed.

Comment (sepandhaghighi) [reply .travis/test.sh]: Where ?

File: .travis/install.sh (hunk 1 above)
Comment (sepandhaghighi) [reply .travis/install.sh]: First I added this line to test `art` library in develop mode, but later removed it ;-)

File: .travis/test.sh (hunk 2 above)
Comment (sadrasabouri) [reply .travis/test.sh]: Nope i might make mistake

**Pull request merged**